src/main/java/uk/gov/companieshouse/chdorderconsumer/service/CHDOrderService.java

```diff
@@ -1,13 +1,18 @@
 package uk.gov.companieshouse.chdorderconsumer.service;
 
+import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Service;
 import uk.gov.companieshouse.api.InternalApiClient;
 import uk.gov.companieshouse.api.error.ApiErrorResponseException;
 import uk.gov.companieshouse.api.handler.exception.URIValidationException;
 import uk.gov.companieshouse.api.model.ApiResponse;
 import uk.gov.companieshouse.api.model.order.chd.MissingImageDeliveryRequestApi;
+import uk.gov.companieshouse.chdorderconsumer.exception.RetryableErrorException;
 import uk.gov.companieshouse.chdorderconsumer.exception.ServiceException;
 
+import static org.springframework.http.HttpStatus.BAD_REQUEST;
+import static org.springframework.http.HttpStatus.UNAUTHORIZED;
```

Comment:
Few unused imports in this file that can be removed

Reply:
Ty fixed all the smelly code